### DISCUSSION TITLE
Issues with building the distro

### DISCUSSION THREAD

**Discussion by JennySimen** [camicroscope/caMicroscope#743] [Q&A]
Hi everyone,
I am facing some issues while running the distro after cloning. This is the snippet of the erro I have. 
<img width="567" alt="Screen Shot 2024-03-12 at 02 16 14" src="https://github.com/camicroscope/caMicroscope/assets/44881714/08eb3e12-a9b4-4c99-a204-c97b9d9ec42c">

**Comment by birm** [comment]
That's certainly interesting. A refused connection from github on https _could_ be due to system clock or vpn issues?
It seems like if you're posting here you're able to connect to github via web. Can you check your clock for accuracy (e.g. in the timezone you expect with the time you expect), let us know if you're using a vpn/firewall/something notable, and/or try again?

**Comment by JennySimen** [comment]
Hi @birm 
I checked and the clock time is accurate and I also do no have any vpn. I will try running the project again to see.
